src/components/FeesField/BitcoinKind.js

```diff
@@ -85,10 +87,10 @@ class FeesField extends Component<Props & { fees?: Fees, error?: Error }, State>
     return { items, selectedItem }
   }
 
-  componentDidUpdate() {
+  componentDidUpdate({ fees: prevFees }: OwnProps) {
     const { feePerByte, fees, onChange } = this.props
     const { items, isFocused } = this.state
-    if (fees && feePerByte.isZero() && !isFocused) {
+    if (fees && fees !== prevFees && feePerByte.isZero() && !isFocused) {
```

Comment:
(note: we only want to force the "reset to default fees" if new fees was loaded and value was still zero. because zero is still allowed)